release-please-config.json

```diff
@@ -16,6 +16,7 @@
     "packages/opentelemetry-id-generator-aws-xray": {},
     "packages/opentelemetry-propagation-utils": {},
     "packages/opentelemetry-test-utils": {},
+    "plugins/node/instrumentation-fs": {},
```

Comment:
based on #921, should it also be added to `.release-please-manifest.json`?

Reply:
I think we don't need that now that release-please is updated to v3

Reply:
Removed that change.